docs/content/releases.md

```diff
@@ -6,6 +6,70 @@ doctypes: ["concept"]
+
+
+IMPROVEMENTS:
+
+* [3170](https://github.com/nginxinc/kubernetes-ingress/pull/3170) Watch subset of namespaces for secrets. Thanks to [Hans Feldt](https://github.com/hafe).
+* [3341](https://github.com/nginxinc/kubernetes-ingress/pull/3341) Set value of `$remote_addr` to client IP when TLSPassthrough and Proxy Protocol are enabled.
+* [3131](https://github.com/nginxinc/kubernetes-ingress/pull/3131) NAP DoS images are now available in the AWS Marketplace.
+* [3231](https://github.com/nginxinc/kubernetes-ingress/pull/3231) Always print build info and flags used at the start.
+* [2735](https://github.com/nginxinc/kubernetes-ingress/pull/2735) Support default client proxy headers to be overwritten in VirtualServer. Thanks to [Alex Wied](https://github.com/centromere)
+* [3133](https://github.com/nginxinc/kubernetes-ingress/pull/3133) Add caseSensitiveHttpHeaders to appolicy crd. Thanks to [Pavel Galitskiy](https://github.com/galitskiy).
+
+
+FIXES:
+
+* [3139](https://github.com/nginxinc/kubernetes-ingress/pull/3139) Remove all IPV6 listeners in ingress resources with `-disable-ipv6` command line.
+
+
+HELM CHART:
+
+* [3113](https://github.com/nginxinc/kubernetes-ingress/pull/3113) Add JSON Schema.
+* [3143](https://github.com/nginxinc/kubernetes-ingress/pull/3143) Add annotations for deployment and daemonset.
+* [3136](https://github.com/nginxinc/kubernetes-ingress/pull/3136) Add controller.dnsPolicy. Thanks to [Dong Wang](https://github.com/wd).
+* [3065](https://github.com/nginxinc/kubernetes-ingress/pull/3065) Add annotations to the service account. Thanks to [0m1xa](https://github.com/0m1xa).
+* [3276](https://github.com/nginxinc/kubernetes-ingress/pull/3276) Add horizontalpodautoscaler. Thanks to [Bryan Hendryx](https://github.com/coolbry95).
+
+
+UPGRADE:
+* Make sure the Kubernetes version is in the supported platforms listed below.
+* For NGINX, use the 3.0.0 images from our [DockerHub](https://hub.docker.com/r/nginx/nginx-ingress/tags?page=1&ordering=last_updated&name=3.0.0), [GitHub Container](https://github.com/nginxinc/kubernetes-ingress/pkgs/container/kubernetes-ingress) or [Amazon ECR Public Gallery](https://gallery.ecr.aws/nginx/nginx-ingress).
+* For NGINX Plus, use the 3.0.0 images from the F5 Container registry or the [AWS Marketplace](https://aws.amazon.com/marketplace/search/?CREATOR=741df81b-dfdc-4d36-b8da-945ea66b522c&FULFILLMENT_OPTION_TYPE=CONTAINER&filters=CREATOR%2CFULFILLMENT_OPTION_TYPE) or build your own image using the 3.0.0 source code.
+* For Helm, use version 0.16.0 of the chart. If you're using custom resources like VirtualServer and TransportServer (`controller.enableCustomResources` is set to `true`), after you run the `helm upgrade` command, the CRDs will not be upgraded. After running the `helm upgrade` command, run `kubectl apply -f deployments/helm-chart/crds` to upgrade the CRDs.
```

Comment:
Instead of "the CRDs will not be upgraded"
Lets describe this clearer with something like a new sentence:
"Helm does not upgrade the CRDs.  After running the 'helm upgrade' command....